app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
Why 2 modules?
module UserHelper
module UserHelper

Reply:
I added it by mistake.